content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -43,11 +43,12 @@ this is true when user namespaces are used.
 * You need to be able to exec into pods
 * Feature gate `UserNamespacesStatelessPodsSupport` need to be enabled.
 
-In addition, support is needed in the
+In addition, support is **needed** in the
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
-to use this feature with Kubernetes stateless pods:
-
-* CRI-O: v1.25 has support for user namespaces.
+to use this feature with Kubernetes stateless pods. You need to meet the
+requirements mentioned in the [concept page](/docs/concepts/workloads/pods/user-namespaces/).
+A list of container runtimes the support user namespaces, and since which
+version they do, is shown there.
 
 Please note that **if your container runtime doesn't support user namespaces, the
 new `pod.spec` field will be silently ignored and the pod will be created without
```

Comment:
:information_source: 
Aside: this ought to change for v1.27 (the field is no longer new, and it's also not `.pod.spec` in any JSON
query language), but this isn't the PR to fix it.

Reply:
what do you mean it is not in pod.spec in any json query language? Can you please elaborate how it is not there?

Reply:
```shell
# doesn't work
kubectl get pods/foo -o jsonpath='{pod.spec}'

# works
kubectl get pods/foo -o jsonpath='{.spec}'
```

Reply:
In the API, `pod` (actually `pods`) is part of the URL, not part of the resource data.

Reply:
Opened a new PR for this, thanks: https://github.com/kubernetes/website/pull/40335